### PR TITLE
Basic html css

### PR DESCRIPTION
**Pull request by saketkum11**
Please review basics setup for notes app

### CODE REVIEW

**Review by shraddha15-creator — APPROVED** [review]
Good folder structure

**Review by JowelTisso — COMMENTED** [review]
I guess you don't need PR review for initial base structure commit. Only feature branch needs review I suppose.

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,43 @@
...
-
+import { Routes, Route } from "react-router-dom";
+import Trash from "./Pages/Trash/Trash";
+import Profile from "./Pages/Profile/Profile";
+import Archieve from "./Pages/Archieve/Archieve";
+import Label from "./Pages/Labels/Labels";
+import Home from "./Pages/Home/Home";
+import Signup from "./Pages/Auth/Signup";
+import Login from "./Pages/Auth/Login";
+import Notes from "./Pages/Notes/Notes";
```
Comment (narharikale) [inline src/App.js:11]: use barrel export here , it would be more readable here

File: src/App.js [hunk 2]
```diff
@@ -1,41 +1,43 @@
...
+          Labels
+        </Route>
+        <Route path="/archieve" element={<Archieve />}>
+          Archieve
+        </Route>
+        <Route path="/trash" element={<Trash />}>
+          Trash
+        </Route>
+        <Route path="/profile" element={<Profile />}>
+          Profile
```
Comment (narharikale) [inline src/App.js:29]: is any reason you are passing text inside these route

**Review by narharikale — APPROVED** [review]

File: src/App.js (hunk 2 above)
Comment (saketkum11) [reply src/App.js:29]: no that is by mistake

**Comment by saketkum11** [comment]
> 
I was thinking that too but after creating PR I realised that.

File: src/App.js (hunk 1 above)
Comment (saketkum11) [reply src/App.js:11]: Haan In previous project u have mentioned this problem .

**Pull request merged**